bundles/org.openhab.core.config.discovery.sddp/src/main/java/org/openhab/core/config/discovery/sddp/SddpDevice.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.config.discovery.sddp;
+
+import java.time.Instant;
+import java.util.Map;
+import java.util.Objects;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * A DTO class containing data from an SDDP device discovery result.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@NonNullByDefault
+public class SddpDevice {
+
+    public final String from;
+    public final String host;
+    public final String maxAge;
+    public final String type;
+    public final String primaryProxy;
+    public final String proxies;
+    public final String manufacturer;
+    public final String model;
+    public final String driver;
+    public final Instant expireInstant;
+
+    /**
+     * Constructor
+     * 
+     * @param headers a map of parameter name / value pairs.
+     */
+    public SddpDevice(Map<String, String> headers) {
+        from = headers.getOrDefault("From", "").replaceAll("^\"|\"$", "");
+        host = headers.getOrDefault("Host", "").replaceAll("^\"|\"$", "");
```

Comment:
Could the host field be further split into a UUID field (the MAC address)? The conventions of this field seems to _Manufacturer-PrimaryProxy-MAC Address_. This will be useful for the creating the ThingUID in a SddpDiscoveryParticipant class.

[SddpDiscoveryParticipant](https://github.com/sammck/sddp-discovery-protocol/blob/main/README.md#servers)

On the one device that I have that spits out SDDP packets, the `Host` field looks like this:
**Host: "Panasonic-dvd-80c755axxxxx"** (mac redacted)

Reply:
Yes. Ok.

Reply:
@mlobstein 4yi `host.split("-")` would work on your Panasonic but the example that you cite above has **Host: "JVC_PROJECTOR-E0DADC152802"** so we probably need to try `host.split("-|_")` .. or??

Reply:
see below